test/fixtures/block_with_class.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.describe(SomeObject)do
```

Comment:
The situation where there are no parens around `SomeObject` needs a SyntaxTree update - see https://github.com/Shopify/ruby-lsp/issues/515.